pkg/storage/tpr/install_types.go

```diff
@@ -34,31 +37,71 @@ var thirdPartyResources = []v1beta1.ThirdPartyResource{
 	serviceBindingTPR,
 }
 
+//Installer takes in a client and exposes method for installing third party resources
```

Comment:
Nit, should probably add a space here (and elsewhere), so it looks like:
// Installer